src/server/handlers.rs

```diff
@@ -1198,4 +1256,28 @@ mod test {
         assert!(!regex.is_match(word_with_period));
         assert!(!regex.is_match(word_with_space));
     }
+
+    #[tokio::test]
+    async fn validate_oam_label_rules() {
+        // Valid labels
+        assert!(valid_oam_label((&"foo".to_string(), &"bar".to_string())));
+        assert!(valid_oam_label((
+            &"app.oam.io/name".to_string(),
+            &"wasmcloud".to_string()
+        )));
+        assert!(valid_oam_label((
+            &"app.oam.io/name".to_string(),
+            &"wasmcloud".to_string()
+        )));
```

Comment:
Are these supposed to be the same?

Reply:
Just to be double sure...

Nope, just be a copy paste fail. Will change one